image/image_linux.go

```diff
@@ -425,7 +425,7 @@ var setupSeed = func(tsto *tooling.ToolingStore, model *asserts.Model, opts *Opt
 			if sn == nil {
 				return "", fmt.Errorf("internal error: downloading unexpected snap %q", info.SnapName())
 			}
-			fmt.Fprintf(Stdout, "Fetching %s\n", sn.SnapName())
+			fmt.Fprintf(Stdout, "Fetching %s(%s)\n", sn.SnapName(), info.SnapRevision())
```

Comment:
nitpick: as we have a real snap.Info this could just be info.Revision

Reply:
I updated this